optaplanner-docs/src/modules/ROOT/pages/integration/config-properties.adoc

```diff
@@ -58,3 +58,17 @@ Terminates the solver when a specific or higher score has been reached.
 For example: `0hard/-1000soft` terminates when the best score changes from `0hard/-1200soft` to `0hard/-900soft`.
 Wildcards are supported to replace numbers.
 For example: `0hard/*soft` to terminate when any feasible score is reached.
+
+{property_prefix}optaplanner.benchmark.solver-benchmark-config-xml::
+A classpath resource to read the solver configuration XML.
```

Comment:
Is that a solver configuration? I thought it was a benchmark configuration.

Reply:
Are we sure we want to write this property name in stone? Clearly, it's ambiguous?

Reply:
Ok, the javadoc API in Quarkus OptaPlannerBenchmarkBuildTimeConfig also has this problem.
And the Spring BenchmarkProperties, which I added it in, has it too.

But it's not a solve configuration, in any of these cases. The naming is confusing.

Reply:
I'll set up a meeting on how to handle this...

Reply:
Meeting decided that we don't need to go further than current state.
@Christopher-Chianelli Would you mind changing this docs line and the javadoc lines for quarkus/spring to from "solver configuration XML" to:
```suggestion
A classpath resource to read the benchmark configuration XML.
```

Reply:
Done; Spring was already benchmark